INSS.EIIR.Functions/FnHealthCheckPing.cs

```diff
@@ -14,12 +14,17 @@ namespace INSS.EIIR.Functions
 {
     public class FnHealthCheckPing
     {
+        private readonly ILogger<FnHealthCheckPing> _logger;
+
+        public FnHealthCheckPing(ILogger<FnHealthCheckPing> logger)
+        {
+            _logger = logger;
+        }
+
         [Function("FnHealthCheckPing")]
         public async Task<IActionResult> Run(
-            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "Health/Ping")] HttpRequestData req,
-            ILogger log)
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "Health/Ping")] HttpRequestData req)
         {
-            log.LogInformation("Health Check Pinged");
             return new OkResult();
```

Comment:
you can use logger to log the information for health check

Reply:
Correct though I have pulled it out if reduce the noise in the logs, as it was, not only would Application Insights log the ping requests - we would also have a "Health Check Pinged" information trace created which is just unnecessary noise - especially if you were interested in other logging  the developer was pumping out.  Of course this makes injecting the logger in the first place irrelevant